pilot/pkg/kube/inject/initializer.go

```diff
@@ -40,6 +41,7 @@ var (
 		{v1.SchemeGroupVersion, &v1.ReplicationController{}, "replicationcontrollers", "/api"},
 
 		{v1beta1.SchemeGroupVersion, &v1beta1.Deployment{}, "deployments", "/apis"},
+		{v1.SchemeGroupVersion, &openshiftv1.DeploymentConfig{}, "deploymentconfigs", "/apis"},
```

Comment:
i think this should be use `openshiftv1.LegacySchemeGroupVersion` instead.In addition,  we also probably want the newer `openshiftv1.SchemeGroupversion` as in the list of supported kinds.

Reply:
Makes total sense. Just updated the code.